### PR TITLE
Typo [QuickFix] - PopupClient.ts

### PR DESCRIPTION
**Pull request by ordonezgs**
Fixing up a Typo of a logger message.

### CODE REVIEW

**Comment by jasonnutter** [comment]
> Thank you for your contribution! Please generate change files by running `npm run beachball:change` from the root of the repo, then answer each prompt and push up the generated files.

@ordonezgs Please see message above to generate and commit changefiles, thanks!

**Review by jasonnutter — CHANGES_REQUESTED** [review]
Changefile needed, otherwise looks good.

**Comment by ordonezgs** [comment]
Done,

Thank you very much, I have already run the above command and generated the corresponding file.

Please let me know of any further requirements.

Thank you very much @jasonnutter .

**Review by ordonezgs — COMMENTED** [review]
Changefile added. Beachball check done.

File: change/@azure-msal-browser-c15c8f41-0fa0-4619-82ff-20addba77243.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "type": "none",
+  "comment": {
+    "title": "",
+    "value": ""
+  },
```
Comment (tnorling) [inline change/@azure-msal-browser-c15c8f41-0fa0-4619-82ff-20addba77243.json]:
```suggestion
  "comment": "Fix typo in logger message #4385",
```

Comment (ordonezgs) [reply change/@azure-msal-browser-c15c8f41-0fa0-4619-82ff-20addba77243.json]: Done, thanks. @tnorling

**Review by tnorling — APPROVED** [review]

**Review by sameerag — APPROVED** [review]

**Pull request merged**